src/utils/json.js

```diff
@@ -44,7 +44,7 @@ const prettyPrint = (json, indentation = '') => {
   } else if (_.isNil(json)) {
     result += 'null';
   } else if (_.isString(json)) {
-    result += `"${json.replace(/"/g, '\\"').replace(/\n/g, '\\n')}"`;
+    result += `"${json.replace(/\\/g, '\\\\').replace(/"/g, '\\"').replace(/\r/g, '\\r').replace(/\n/g, '\\n')}"`;
```

Comment:
- Since the replace part is not easy to read, it could be explained with a notice
- It's hard to read on one line, we could multiline it.
- On a side note, this escapement is not complete. Some strings will still fail (see screenshot). According to this site: https://www.json.org/json-en.html, you have to escape more characters.


Here is a suggestion:

```suggestion
    // NOTICE: Escape invalid characters (see: https://www.json.org/json-en.html)
    const escapedJsonString = json
      .replace(/[\\]/g, '\\\\')
      .replace(/[\"]/g, '\\\"')
      .replace(/[\/]/g, '\\/')
      .replace(/[\b]/g, '\\b')
      .replace(/[\f]/g, '\\f')
      .replace(/[\n]/g, '\\n')
      .replace(/[\r]/g, '\\r')
      .replace(/[\t]/g, '\\t');
    result += `"${escapedJsonString}"`;
```

Screenshot:

<img width="470" alt="Capture d’écran 2020-01-14 à 19 09 58" src="https://user-images.githubusercontent.com/1575946/72370601-e6cedb80-3702-11ea-80d6-88dab63ef403.png">

Reply:
I guess I was a little bit to focused on the `\r` issue. Thanks for pointing me in that direction !
`\u` is a nightmare to handle (Since \u can represent any char, like `\u000A` is `\n`). Maybe that's why you didn't include it in the code sample.
I just had to fix lint issue related to useless char escape (like `"`), but i clearly think this solution is way better than mine!